SoftLayer/managers/ordering.py

```diff
@@ -745,3 +746,14 @@ def delete_quote(self, quote_id):
         """
 
         return self.client['SoftLayer_Billing_Order_Quote'].deleteQuote(id=quote_id)
+
+    def get_all_cancelation(self):
+        """returns the all cancelations, completed orders"""
+
+        mask = 'mask[id,itemCount,modifyDate,createDate,ticketId,' \
+               'ticket[assignedUserId,id,attachedHardware[id,hostname,domain],' \
+               'attachedVirtualGuests[id,hostname,domain],' \
+               'attachedDedicatedHosts[id,name],serviceProviderResourceId],' \
+               'status[name,id],user[id,firstName,lastName],' \
+               'items[billingItem[cancellationDate,categoryCode,pendingCancellationFlag]]]'
+        return self.client.call('SoftLayer_Billing_Item_Cancellation_Request', 'getAllCancellationRequests', mask=mask)
```

Comment:
Add a limit of 50 here, for this method the default is only 25 which I don't think is quite enough. Showing the last 50 cancellations should be good though.